### ISSUE TITLE
Highlight color unreadable

### ISSUE THREAD

**Issue by mbernasocchi** [nana-4/Flat-Plat#150]
Hi the higlight color is unreadable (white on white) see screenshots below
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

If you report a bug or ask a question, please include the following information as required:
- Flat-Plat version:  Flat-Plat-dark-compact from github
- GTK+ 3 version (You can check it with `pkg-config --modversion gtk+-3.0`)
- Distribution (and version): Ubuntu 17.04
- Desktop environment (and version): GNOME Shell 3.24.2
- Related application (and version): pgAdmin III 1.22.2, gedit - Version 3.22.0
- Screenshot(s)

![image](https://user-images.githubusercontent.com/233663/30650958-982d64ca-9e24-11e7-86ba-b3c71818c637.png)
![image](https://user-images.githubusercontent.com/233663/30652335-6c3f791c-9e28-11e7-90f7-2ca10a07ce72.png)

**Comment by nana-4** [comment]
I think these are bugs of each application. It should appear similarly in other dark themes.

Example of gedit with _Adwaita-dark_:
![image](https://user-images.githubusercontent.com/9556384/30658528-f14c1f8c-9e75-11e7-8b2f-2c812af2d256.png)

I don't think these bugs can be fixed by themes. These bugs should be reported to the bug tracker of each application, I guess.

**Comment by nana-4** [comment]
Hmm, I don't know how about pgAdmin III, but for gedit (actually GtkSourceView) it may be possible to deal with this theme.

**Comment by mbernasocchi** [comment]
in gedit there is a dark theme that works. its called oblivion iirc

**Comment by nana-4** [comment]
Yeah, I'm considering support GtkSourceView theme(s) like oblivion for gedit.

**Comment by OriginalFannibal** [comment]
Here are a few Gedit themes that you could use as a base or for reference. 
- [Material theme for Gedit editor](https://github.com/systematicat/gedit-material-theme)
- [Gedit Material Theme](https://github.com/systematicat/gedit-material-theme)

I'm using the first presently and it looks pretty good but having something that was designed for Materia would be better :)
